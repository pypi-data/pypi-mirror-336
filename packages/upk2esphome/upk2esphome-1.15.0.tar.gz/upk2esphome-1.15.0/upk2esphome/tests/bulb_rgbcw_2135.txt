{Jsonver:1.1.9,brightmax:95,brightmin:1,cagt:20,category:0505,cmod:rgbcw,colormax:100,colormin:10,crc:23,cwmaxp:100,cwtype:0,defbright:100,defcolor:c,deftemp:100,dmod:3,ehccur:6,ehwcur:3,gmkb:60,gmkg:60,gmkr:80,gmwb:75,gmwg:70,gmwr:100,iicb:0,iicc:4,iicg:1,iicr:2,iicscl:7,iicsda:9,iicw:3,module:WBLC9,onoffmode:1,pmemory:1,prodagain:0,remdmode:0,rgbt:10,rstbr:50,rstcor:c,rstnum:3,rsttemp:100,title20:1,wfcfg:spcl,wfct:3,wt:20,}
