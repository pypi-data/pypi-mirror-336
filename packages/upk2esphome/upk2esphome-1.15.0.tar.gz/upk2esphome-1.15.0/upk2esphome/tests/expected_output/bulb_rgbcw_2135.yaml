sm2135:
  clock_pin: P7
  data_pin: P9
  rgb_current: 40mA
  cw_current: 25mA

output:
  - platform: sm2135
    id: output_red
    channel: 2
  - platform: sm2135
    id: output_green
    channel: 1
  - platform: sm2135
    id: output_blue
    channel: 0
  - platform: sm2135
    id: output_cold
    channel: 4
  - platform: sm2135
    id: output_warm
    channel: 3

light:
  - platform: rgbww
    id: light_rgbww
    name: Light
    color_interlock: true
    cold_white_color_temperature: 6500 K
    warm_white_color_temperature: 2700 K
    red: output_red
    green: output_green
    blue: output_blue
    cold_white: output_cold
    warm_white: output_warm
